### PR TITLE
Minor repository cleanup

### PR DESCRIPTION
**Pull request by pmderodat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
 [![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/ada-lang/Alire)
 
-Default index for the `alr` tool.
-
-These are community-maintained Ada projects, ready to use.
-
-Please visit https://github.com/alire-project/alire to download `alr`.
+This repository hosts the official community-maintained index of Ada projects
+to used with the `alr` tool. Please visit
```
Comment (mosteo) [inline README.md]: "to used" -> "to be used"

File: README.md [hunk 2]
```diff
@@ -5,3 +5,15 @@
 This repository hosts the official community-maintained index of Ada projects
 to used with the `alr` tool. Please visit
 https://github.com/alire-project/alire to download `alr`.
+
+## TODO
+
+Here are sources of projects that might be interesting to add to this index:
```
Comment (mosteo) [inline README.md:11]: Right now we have also three issues listing possible additions (#29, #30, #31). Just a reminder that at some point we could centralize this information (not necessarily in this PR).

**Review by mosteo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pmderodat) [reply README.md]: Thanks, will fix!

File: README.md (hunk 2 above)
Comment (pmderodat) [reply README.md:11]: Ok, thanks! Not sure what else to do in this README (mentionning these issues?), so I’ll just leave this list as it is for now.

**Pull request merged**